### DISCUSSION TITLE
docs/stable/advance/with-openstack/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#4798] [Announcements]
# docs/stable/advance/with-openstack/

None

https://kubeovn.github.io/docs/stable/advance/with-openstack/

**Comment by acechef** [comment]
共享底层 OVN方式，为什么我在改了ml2_conf.ini和OVS 配置后，在 Kubernetes 侧，查询不到同步过来的VPC 资源？有什么排查方法吗

**Reply by zbb88888** [reply]
`从 OpenStack 同步过来的 VPC 资源`: 可能是 openstack 那边要做改动？

**Reply by acechef** [reply]
已经在openstack侧根据文档调整了ml2_conf配置和ovs配置了

**Reply by zbb88888** [reply]
openstack 可能他们自己改代码了？

**Reply by acechef** [reply]
可以了，是我配置有点问题
